README.md

```diff
@@ -0,0 +1,252 @@
+# ACCESS-ESM with PAYU
+
+## Understanding PAYU
+
+PAYU was designed to help users of the NCI system run climate models.
+It was initially created for MOM, but has been adapted for other models,
+including coupled ones.
+
+The aim of PAYU is to make it easy and intuitive to configure and run the models.
+
+PAYU knows certain models and how to run them. Adding more models needs additions to the PAYU sources.
+This will not be part of this document.
+
+### Terms
+
+To understand PAYU, it helps to distinguish certain terms:
+
+-   The **Laboratory** is a directory where all parts of the model are kept.
+    It is typically in the user's short directory, usually at `/short/$PROJECT/$USER/<MODEL>`
+-   The **Control Directory** is the directory where the model configuration is
+    kept and where from where the model is run.
+-   The **work** directory is where the model will actually be run.
+    It is typically a subdirectory of the Laboratory.
+    Submodels will have their own subdirectories in the work directory, named
+    after their name in the master configuration file.
+-   The **archive** directory is where payu pushes all output files after each run.
+
+The **work** and **archive** directories will be automatically created by payu, the
+other two you will have to create.
+
+### The master configuration file
+
+In the Control Directory, the file `config.yaml` is the master control file.
+Examples of what is configured in this file are:
+
+-   The actual model to run.
+-   Where to find the model binaries and configurations
+-   What resources to request from the scheduling system (PBS)
+-   Links to the laboratory
+-   Start date and run length per submission pf the model
+
+The model configuration files are typically in subdirectories of the Control Directory,
+the location of which is referenced in the master control file.
+Since the models itself do need different ways to set up the model, the contents of these subdirectories will differ between different models.
+
+## Understanding ACCESS-ESM
+
+ACCESS (Australian Community Climate and Earth System Simulator) is a Coupled Climate Model.
+
+The ESM 1.5 subversion of ACCESS specifically contains these models:
+
+| Component  | Model      | Version |
+| ---------- | ---------- | ------- |
+| Atmosphere | UM-HG3     | 7.3     |
+| Ocean      | MOM        | 4       |
+| Sea Ice    | CICE       | 4.1     |
+| Coupler    | OASIS-MCT  | 3.5     |
+
+Pre-compiled executables for these models are available on raijin at
+`/short/public/access-esm/payu/bin/csiro/`.
+
+## Setting up ACCESS-ESM with PAYU
+
+### The pre-conditions
+
+On `raijin`, first make sure that you have access to our modules.
+This can most easily been done by adding the line
+
+    module use /g/data3/hh5/public/modules
+
+to your `~/.bashrc`, then logging back in. Then all you have to do is
```

Comment:
I don't think this will be the way we will do this. `payu` will just be part of `conda/analysis3`. Initially only available in `conda/analysis3-unstable`